### ISSUE TITLE
SpawnSpheres are not automtically put in the according SimGroup

### ISSUE THREAD

**Issue by Duion** [GarageGames/Torque3D#1947]
While trying to change the default folder the SpawnSpheres are in I found:
https://github.com/GarageGames/Torque3D/blob/development/Templates/Full/game/tools/worldEditor/gui/objectBuilderGui.ed.gui#L954
This checks when creating a new SpawnSphere, if a SimGroup called "PlayerDropPoints" exists and if not it creates it.
After that "%this.objectGroup = "PlayerDropPoints";" is probably supposed to automatically make the newly created SpamSphere being put in the newly created SimGroup, but it actually does not work, the new SpamSphere is created in the root MissionGroup folder.

So is this broken or did I get something wrong?

**Comment by dottools** [comment]
You're not wrong, it is suppose to be placed into `PlayerDropPoints` SimGroup according to this code:
https://github.com/GarageGames/Torque3D/blob/development/Templates/Full/game/tools/worldEditor/gui/objectBuilderGui.ed.gui#L525-L526
